infra/aws/terraform/prow-build-cluster/modules/iam/users.tf

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+data "aws_iam_user" "user_xmudrii" {
```

Comment:
How about replacing this with a loop iterating over `maintainers_list` variable? That would simplify adding new maintainers.

Reply:
I used a local variable for this because we need to query the AWS API for ARNs, so a classic variable in `terraform.tfvars` is not enough.

Reply:
https://github.com/kubernetes/k8s.io/pull/5113/files/c70c8c2198a01272e2e6d702848f430f0cd46dce#r1175115625